pwndbg/commands/context.py

```diff
@@ -801,11 +800,14 @@ def context_backtrace(with_banner=True, target=sys.stdout, width=None):
     bt_prefix = "%s" % pwndbg.gdblib.config.backtrace_prefix
     while True:
         prefix = bt_prefix if frame == this_frame else " " * len(bt_prefix)
-        prefix = f" {c.prefix(prefix)}"
-        addrsz = c.address(pwndbg.ui.addrsz(frame.pc()))
-        symbol = c.symbol(pwndbg.gdblib.symbol.get(int(frame.pc())))
+        prefix = " %s" % c.prefix(prefix)
+        frame_pc = frame.pc()
+        stackaddr = M.get((pwndbg.gdblib.stack.find_addr_on_stack(frame_pc)))
```

Comment:
It's a little hacky, but to deal with the alignment issue you brought up [here](https://github.com/pwndbg/pwndbg/issues/1288#issuecomment-1595693469) we can call the [`M.get`](https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/color/memory.py#L19) function with the optional `text` parameter. We can convert the address to a hex string that's already padded to a width of `pwndbg.gdblib.arch.ptrsize`, use that as the `text` argument, and let `M.get` color it properly.

Also nit: seems like there are extra parenthesis here you can remove.

Reply:
done